### PR TITLE
added translateUnsafeHTML directive

### PR DESCRIPTION
**Pull request by stefanholzapfel**
Hi!

I have written a translateUnsafeHTML directive as needed in issue #13 and in the way proposed by @davidsteinberger in issue #9 .

Please review and merge if it's viable.

### CODE REVIEW

File: src/lib/directive.ts [hunk 1]
```diff
@@ -64,3 +64,17 @@ export const langChanged = directive((cb: LangChangedDirectiveCallback) => (part
...
+ * @param key
+ * @param values
+ * @param config
+ */
+export const translateUnsafeHTML = (key, values?, config?) => langChanged(() => {
+	const tl = get(key, values, config);
+	const template = document.createElement('template');
+	template.innerHTML = tl;
+	return document.importNode(template.content, true);
+});
```
Comment (andreasbm) [inline src/lib/directive.ts]: Looks good 👍  I have two suggestions for the implementation.

1. We use the `unsafeHTML` function from `lit-html/directives/unsafe-html` to get the benefits of the [`lit-html` implementation](https://github.com/Polymer/lit-html/blob/master/src/directives/unsafe-html.ts).

2. We change the name of the function to `unsafeTranslate` to stay consistent with the naming of the `unsafeHTML` function.

The suggestions are shown in the code below.

```js
export const unsafeTranslate = (key: string, values?: Values | ValuesCallback, config?: ITranslateConfig) => langChanged(() => unsafeHTML(get(key, values, config)));
```

**Comment by stefanholzapfel** [comment]
Hi Andreas!

Thanks for your feedback.
Sure, using the original unsafeHTML directive is much smarter :)

I have made another change and added a parameter to the original translate instead of writing it's own directive. What do you say?

Of course we could also move the renderHTML bool into the ITranslateConfig interface if you prefer.

**Comment by stefanholzapfel** [comment]
@tree shaking: Good point, reverted to 2 functions.

I'm not happy with the naming though. Since the translations stem from a json file that is under the developers control, it's not "unsafe" to use in the original sense of unsafeHTML where "unsafe" should raise awareness that sanitization is necessary for user input. The term "HTML" in contrast adds information about what differentiates both directives.

But I now know what it does, so you can merge it that way if you want :)

**Comment by andreasbm** [comment]
> I'm not happy with the naming though. Since the translations stem from a json file that is under the developers control, it's not "unsafe" to use in the original sense of unsafeHTML where "unsafe" should raise awareness that sanitization is necessary for user input. The term "HTML" in contrast adds information about what differentiates both directives.

I think it's a very valid point - I might revert the name back to `translateUnsafeHTML` before release 👍 Thanks a lot for your PR - it's a very nice addition to lit-translate!

I will merge it now, have a great weekend.

**Pull request merged**